site/pmguides/src/php_laravel_API/laravel.md

```diff
@@ -0,0 +1,243 @@
+author: Greg Bulmash
+id: php_laravel_API
+summary: Create A REST API with PHP And Laravel 
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/loopDelicious/pmquickstarts
+tags: Getting Started, Developer, Tester, Automation, PHP 
+
+# Create A REST API with PHP And Laravel
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+PHP is the programming language behind major web applications like WordPress (blog), Drupal ([CMS](https://en.wikipedia.org/wiki/Content_management_system)), and Magento (e-commerce). It first appeared in 1995, less than 3 weeks after the Java programming language.
+
+It remains one of the top 10 programming languages in the world, according to many lists, and is the "P" in the famous LAMP stack (Linux, Apache, MySQL, and PHP).
+
+In this tutorial, let's create a simple API using one of PHP's most popular frameworks, [Laravel](https://laravel.com/), and then access it with Postman. 
+
+### Prerequisites
+- PHP
+- Composer for PHP
```

Comment:
Since these are already covered in the "What You'll Need" section, it might be confusing to have them here as well. The prereqs can be a little more general--something like familiarity with Postman, APIs, or PHP.